.devcontainer/get-nunit-apidocs.sh

```diff
@@ -1,15 +1,15 @@
 # This script gets the the latest version of NUnit and extracts it. This is because docfx is capable of generating API docs, and this way the Codespace will be automatically enabled to do that.
 
 # This is hard-coded for now.
-NUNIT_VERSION_FOR_API_DOCS="4.0.0"
+NUNIT_VERSION_FOR_API_DOCS="4.3.2"
 
 first_workspace="$(cd /workspaces && ls | head -1)"
 
-wget "https://github.com/nunit/nunit/releases/download/v$NUNIT_VERSION_FOR_API_DOCS/NUnit.Framework-$NUNIT_VERSION_FOR_API_DOCS.zip" -O /apidocs.zip
+wget "https://github.com/nunit/nunit/releases/download/$NUNIT_VERSION_FOR_API_DOCS/NUnit.Framework-$NUNIT_VERSION_FOR_API_DOCS.zip" -O /apidocs.zip
```

Comment:
ℹ️ Updated because recent NUnit tags haven't used the `v` prefix, which they used to.